target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -3,6 +3,10 @@
 PART_NAME=firmware
 REQUIRE_IMAGE_METADATA=1
 
+get_magic_long_vr2600v() {
```

Comment:
Please do:

```
$ git remote add upstream https://git.lede-project.org/source.git
$ git pull --rebase upstream master
```

To pull in the latest commits from LEDE to test the sysupgrade.